newIDE/app/src/EventsSheet/index.js

```diff
@@ -124,6 +127,7 @@ type ComponentProps = {|
   ...Props,
   authenticatedUser: AuthenticatedUser,
   preferences: Preferences,
+  leaderboards: LeaderboardState,
```

Comment:
`leaderboards` doesn't seem to be the best name but hard to find another one: `leaderboardManager`? `leaderboardHandler`?

Reply:
This prop should be optional I think, when used in an External function, there is no notion of project/gameId. There might be a bug when opening an extension and editing its events sheets